### ISSUE TITLE
No password on registration

### ISSUE THREAD

**Issue by IanMiddlebrook** [BiologicalRecordsCentre/UKBMS-online#51]
Hi, the current email to new registrations is as follows:

_RegistTrial,

Thank you for registering at UKBMS. You may now log in to http://www.ukbms.org/mydata/user using the following username and password:

username: RegistTrial
password:

You may also log in by clicking on this link or copying and pasting it in your browser:

http://www.ukbms.org/mydata/user/reset/6151/1502460036/ZBJ4MXVJuE1bLqOiwONW1XIn85RWk3eIcY80WoqwaXg

This is a one-time login, so it can be used only once.

After logging in, you will be redirected to your view account details page.
You should then visit your account's edit page at http://www.ukbms.org/mydata/user/6151/edit so you can change your password.

--  UKBMS team_

**The password is blank.**
If we do not issue a password on registration, then can we remove that part of the message, and just leave the link. 
IE. "Thank you for registering with www.ukbms.org/mydata. You may now log in to your account by clicking on this link or copying and pasting it in your browser:" etc.

Otherwise, if we are supposed to be issuing a password, then we need to make sure that happens and the field is not left blank.

Thanks,
Ian

**Comment by Gary-van-Breda** [comment]
I have removed the reference to passwords on the initial sentence and under the stated username.

**Comment by DavidRoy** [comment]
@IanMiddlebrook are you able to confirm this has been resolved?

**Comment by DavidRoy** [comment]
@IanMiddlebrook bump

**Comment by IanMiddlebrook** [comment]
Hi @DavidRoy 
I've just tested this by registering a new account.
I was amazed to receive an email with the following message:
"Thank you for applying for an account. Your account is currently pending approval by the site administrator.
Once it has been approved, you will receive an e-mail containing further instructions."

I then checked the list of Users and found that all the most recent members were blocked and, presumably, awaiting my approval.

When did this change happen, and did nobody think to tell me?

Anyway, having unblocked my new account, I then received a second email which was fine.

**Comment by DavidRoy** [comment]
Sorry, likely an unintended consequence of stopping/starting the scheme :-(

We'll re-instate asap

Did you test on ukbms.org/mydata?
or the test site?

**Comment by IanMiddlebrook** [comment]
on the live site at ukbms.org/mydata

**Comment by Gary-van-Breda** [comment]
There are complications around the registration of users during the migration period to Drupal 8.

**Comment by IanMiddlebrook** [comment]
well that's fine, as long as I know that I need to approve new users.

**Comment by DavidRoy** [comment]
@Gary-van-Breda having discussed with @IanMiddlebrook, we'll continue to allow registrations/allocations on the drupal 7 (www.ukbms.org/mydata) site.  When the drupal 8 site is ready we'll then need to do another user migration (full or partial) before completing the switch.

Can you turn the relevant functionality back on for www.ukbms.org/mydata and then close this issue

**Comment by Gary-van-Breda** [comment]
www.ukbms.org/mydata changed so that visitors can register themselves.
I'll raise another issue to capture the user migration for D8
